### PR TITLE
Add new payment method: Australian PayID

### PR DESCRIPTION
**Pull request by ripcurlx**
See #4078 for details.

I had to create my own branch as I wasn't able to push to the PR and I had to cherry-pick the original commit as it wasn't signed by @mercurytoxic.

Tested it on Regtest and works still with the old test trade on Mainnet.

### CODE REVIEW

**Review by sqrrm — APPROVED** [review]
utACK

File: core/src/main/java/bisq/core/payment/AustraliaPayid.java [hunk 1]
```diff
@@ -17,53 +17,37 @@
...
-    public String getPayID()
-    {
-        return ((AustraliaPayIDPayload) paymentAccountPayload).getPayid();
+    public String getPayid() {
+        return ((AustraliaPayidPayload) paymentAccountPayload).getPayid();
     }
-    public void setPayID(String payid)
-    {
+
+    public void setPayid(String payid) {
```
Comment (chimp1984) [inline core/src/main/java/bisq/core/payment/AustraliaPayid.java:40]: Our naming convention would be: `setPayId`  and `payId`
Would require also changes in the protobuf

**Comment by chimp1984** [comment]
Please add the an "AUSTRALIA_PAYID" entry in TradeStatistics3.PaymentMethodMapper enum

**Review by chimp1984 — CHANGES_REQUESTED** [review]

File: core/src/main/java/bisq/core/payment/AustraliaPayid.java (hunk 1 above)
Comment (sqrrm) [reply core/src/main/java/bisq/core/payment/AustraliaPayid.java:40]: I think payid is one word in this case. It's not an id as part of bisq, just the name of the payment method.

Comment (chimp1984) [reply core/src/main/java/bisq/core/payment/AustraliaPayid.java:40]: True. They use PayID... anyway not sure whats best here, but also not really important...

**Comment by ripcurlx** [comment]
> Please add the an "AUSTRALIA_PAYID" entry in TradeStatistics3.PaymentMethodMapper enum

It was already added by you as there was a live trade on Mainnet by the dev 😉

File: core/src/main/java/bisq/core/payment/AustraliaPayid.java (hunk 1 above)
Comment (ripcurlx) [reply core/src/main/java/bisq/core/payment/AustraliaPayid.java:40]: I had it like that before, but I agree with @sqrrm that it is actually a brand name and should be without camelcase in that case.

**Comment by chimp1984** [comment]
> > Please add the an "AUSTRALIA_PAYID" entry in TradeStatistics3.PaymentMethodMapper enum
> 
> It was already added by you as there was a live trade on Mainnet by the dev 😉

Ah, true ;-)

**Pull request merged**